ff_stdio.c

```diff
@@ -2179,9 +2178,7 @@ int prvFFErrorToErrno( FF_Error_t xError )
         xTime.tm_mon = pxTime->Month - 1;
         xTime.tm_year = pxTime->Year - 1900;
 
-        xReturn = FreeRTOS_mktime( &xTime );
-
-        return xReturn;
+        return FreeRTOS_mktime( &xTime );
```

Comment:
Here I prefer the two steps: assign to a variable and return it, so it can be inspected while debugging.
An optimising compiler will remove the extra step.

Reply:
Not all compilers will remove the extra step especially if the same value is used above.  And if you want to look at the value you can just break on the `return` line, enter the `FreeRTOS_mktime()`.  and check the value there.  Or where it is consumed in the function above.